.github/workflows/docker.yaml

```diff
@@ -0,0 +1,36 @@
+name: Create and publish a Docker image
+
+on:
+  release:
+    types: [published]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```

Comment:
Why the hash after the @? I would expect some version number on these actions

Reply:
No real reason, just used what was recommended from GitHub documentation page:
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages

```
# GitHub recommends pinning actions to a commit SHA.
# To get a newer version, you will need to update the SHA.
```

I'll make change to update to versions instead of commit SHA.